hack/make-rules/test-cmd-util.sh

```diff
@@ -1574,6 +1574,30 @@ __EOF__
   # Post-Condition: assertion object exist
   kube::test::get_object_assert customresourcedefinitions "{{range.items}}{{$id_field}}:{{end}}" 'bars.company.com:foos.company.com:'
 
+  kubectl "${kube_flags_with_token[@]}" create -f - << __EOF__
```

Comment:
If you have to do another push, add a comment here explaining why this test is important